Adds new now init to cra guide (#548)
zeit/docs: pages/guides/deploying-react-with-now-cra.mdx

Once the app is deployed, you will receive a deployment URL similar to the following: https://my-cra-project-4rx7b16z3.now.sh

## Deploying a new CRA project using the Now CLI

### Pre-requisites:

> - `now` v14.0.0 and above

Reviewer: ```suggestion
The method of initializing a project with Now requires Now CLI version 13.0.0 or above.

If you need to install Now CLI and haven't followed the guide above, you can install Now CLI via [Now Desktop](/docs/v2/getting-started/installation/#now-desktop) for automatic updates or [directly](/docs/v2/getting-started/installation/#now-cli).
```

https://github.com/zeit/now-cli/releases/tag/13.0.0
We added the command in 13.0.0 not 14.0.0

> If you have not yet installed Now, you can do so by installing the [Now Desktop app](/docs/v2/getting-started/installation/#now-desktop) which installs Now CLI automatically, or by [installing Now CLI](/docs/v2/getting-started/installation/#now-cli) directly.

Reviewer: This seems like a direct copy and paste of line 43. Perhaps we can use the above `prerequisites` section to ask the user to install it. I'll add a change request there.

> We'll be using the CLI throughout this guide.

Reviewer: This doesn't seem necessary since the above section sets this up.

<TerminalOutput>
$ now -v <br/>
14.0.2</TerminalOutput>
<Caption>`now -v` will show you your currently installed now version.</Caption>

> The easiest way to create and deploy a new project with CRA is to use the templates provided by the now cli.

Reviewer: ```suggestion
The easiest way to create and deploy a new project with CRA is to use the ready-to-deploy example templates provided by the ZEIT.
```

<TerminalOutput>

> $ now init <br/>

Reviewer: I think we can separate this in a different `<TerminalInput>` component above this, then describe the output below with the TerminalOutput component.

> Select example: (Use arrow keys) <br/>
  apollo <br/>
  bash <br/>
❯ create-react-app <br/>
  docz <br/>
  express <br/>
  gatsby <br/>
  go <br/>
  gridsome<br/> 
  html-minifier<br/> 
  mdx-deck <br/>
  mdx-deck-advanced <br/>
  monorepo <br/>
  nextjs <br/>
  nextjs-news <br/>
  nextjs-static <br/>
(Move up and down to reveal more choices)</TerminalOutput>
<Caption>`now init` will give you a list of projects you can choose, for this guide we're using the `create-react-app` example</Caption>

This will make a new directory called `create-react-app` with everything configured out of the box.

> Run `now` from the root of the project and you'll get a link to the deployed project.

Reviewer: Perhaps we should say "To learn more about what's included in this premade project, read this guide from the start"?

<Image
  src={`${
    process.env.ASSETS

>   }/guides/deploying-react-with-now-cra/running-cra-with-now.png`}

Reviewer: Would you mind making this screenshot a bit smaller? Our standard size is, since the image is oversized, 740px but taken on a retina screen so it ends up being 1480px in width but rendered like it's 740px in width. If that makes sense?

Author: So just resize it to 740px width?

  width={1852 / 2.5}
  height={1182 / 2.5}

>   oversize

Reviewer: Let's make the image a bit smoother with a borderRadius 🙏 
```suggestion
  oversize
  borderRadius
```
